app/views/service_providers/_form.html.erb

```diff
@@ -224,7 +224,9 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
   <div class='saml-fields'>
     <%= form.input :acs_url,
                    input_html: { class: 'usa-input' },
+                   label_html: { class: 'usa-input-required'},
                    label: 'Assertion Consumer Service URL',
+                   required: true,
```

Comment:
is setting `required` initially an issue if a user doesn't select a different protocol? IE, `toggleFormFields` is never triggered? (or does the function trigger when the page is loaded)

Reply:
I will check.

Reply:
ok, so I think I would want to remove this `required` because if someone has JS disabled, they would not be able to submit the form, even if they chose OIDC protocol.  
The other `required` fields are universally required.

Reply:
that makes sense to me

Reply:
So, that doesn't work with Ruby, and the way we add the `required` bug. I'm going to leave this as is.